data-prepper-plugins/opensearch-source/src/main/java/org/opensearch/dataprepper/plugins/source/opensearch/worker/client/model/SearchPointInTimeResponse.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.source.opensearch.worker.client.model;
+
+import org.opensearch.dataprepper.model.event.Event;
+
+import java.util.List;
+
+public class SearchPointInTimeResponse {
```

Comment:
I'd perhaps rename this. At first glance it looks like it is intended to be similar to the opensearch-java APIs. But, it is quite different.

Perhaps: `SearchPointInTimeResults` or `SearchPointInTimeOutput`?